### PR TITLE
[WIP] Add uncollectif for kubevirt providers

### PR DESCRIPTION
**Pull request by valaparthvi**
This PR brings the following changes:
1. Mark tests with `uncollectif` whenever `kubevirt` provider is encountered.

{{ pytest: cfme/tests/cloud_infra_common/test_custom_attributes_rest.py cfme/tests/cloud_infra_common/test_power_control_rest.py cfme/tests/cloud_infra_common/test_rest_providers.py  cfme/tests/intelligence/reports/test_canned_corresponds.py cfme/tests/infrastructure/test_providers.py --use-template-cache -sqvvv --use-provider=kubevirt --long-running }}

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_custom_attributes_rest.py [hunk 1]
```diff
@@ -124,7 +125,8 @@ def _uncollectif(appliance, provider, collection_name):
     return (
         (appliance.version < '5.9' and collection_name in COLLECTIONS_ADDED_IN_59) or
         (provider.one_of(InfraProvider) and collection_name == 'instances') or
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_custom_attributes_rest.py]: Required: 
Please remove check: ```appliance.version < '5.9'```

File: cfme/tests/cloud_infra_common/test_rest_providers.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 from cfme.cloud.provider import CloudProvider
 from cfme.infrastructure.provider import InfraProvider
 from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
 from cfme.utils.blockers import BZ
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_rest_providers.py]: ```from cfme.infrastructure.provider.kubevirt import KubeVirtProvider``` should be on line 7

File: cfme/tests/infrastructure/test_providers.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 from cfme.infrastructure.provider.rhevm import RHEVMProvider, RHEVMEndpoint
 from cfme.infrastructure.provider.virtualcenter import VMwareProvider, VirtualCenterEndpoint
 from cfme.infrastructure.provider.scvmm import SCVMMProvider
+from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
 from cfme.utils.appliance.implementations.ui import navigate_to
```
Comment (ganeshhubale) [inline cfme/tests/infrastructure/test_providers.py]: same here about ```from cfme.infrastructure.provider.kubevirt import KubeVirtProvider```. It should be above ```from cfme.infrastructure.provider.scvmm import SCVMMProvider```

File: cfme/tests/infrastructure/test_providers.py [hunk 4]
```diff
@@ -214,6 +215,7 @@ def test_api_port_max_character_validation_infra(appliance):
 
 @pytest.mark.rhv1
 @pytest.mark.usefixtures('has_no_infra_providers')
+@pytest.mark.uncollectif(lambda provider: provider.one_of(KubeVirtProvider))
 @pytest.mark.meta(blockers=[BZ(1559796, forced_streams=['5.8', '5.9'],
     unblock=lambda provider: not provider.one_of(RHEVMProvider))])
```
Comment (ganeshhubale) [inline cfme/tests/infrastructure/test_providers.py]: Required:
Remove stream: ```5.8```
Also 1559796 is [fixed](https://bugzilla.redhat.com/show_bug.cgi?id=1562788#c7) for 5.9 too. So we can remove this mark.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by valaparthvi** [comment]
@john-dupuy I'm closing this PR since we're removing kubevirt provider support and these changes will no longer be required.

**Pull request closed without merging**